imagersite/imager_images/forms.py

```diff
@@ -0,0 +1,43 @@
+from .models import *
+from django.forms import ModelForm
+from django.contrib.gis import forms as geoforms
+# import floppyforms as geoforms
+
+class AlbumForm(ModelForm):
+    class Meta:
+        model = Album
+        fields = ['title', 'description', 'privacy', 'cover', 'photos']
+
+
+class PhotoForm(ModelForm):
+    class Meta:
+        model = Photo
+        fields = ['title', 'description', 'privacy', 'file']
```

Comment:
Why not include the location field directly here?  Do you need the separate form below?